package.json

```diff
@@ -81,7 +82,6 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.6.0",
-    "faker": "^4.1.0",
```

Comment:
I think you mean regular dependencies here, and I think it makes sense as its used in the actually execution of the code vs something like `jest` or `eslint` which is not required for execution